client/src/app/component/production/production-content.directive.html

```diff
@@ -26,8 +26,8 @@
             à appliquer des formats en utilisant des caractères spéciaux (* = * [] !).</label>
                     <textarea class="form-control" rows="10" ng-model="ctrl.production.content" id="content" name="content" ng-disabled="ctrl.isLocked"
                               ng-required="true"
-                              ng-maxlength="3000"></textarea>
-        <small ng-if="!frm.content.$error.maxlength">({{3000 - ctrl.production.content.length }}
+                              ng-maxlength="5000"></textarea>
```

Comment:
Is it not to high ? 
But If the users need this width, there's no problem, we use a clob in database. 
So if you want to merge let's go